libexec/bats-core/bats-format-junit

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
```

Comment:
I don't know if it's in here, but I'm seeing 25-30s pauses in this formatter when tailing the output file. Can you reproduce?

```bash
$ time bats --formatter bats-format-junit --filter "errors are caught" test
1..2
ok 1 halfway setup_file errors are caught and reported
ok 2 halfway teardown_file errors are caught and reported

real    0m31.577s
user    0m22.649s
sys     0m12.468s

$ time bats --filter "errors are caught" test
 ✓ halfway setup_file errors are caught and reported
 ✓ halfway teardown_file errors are caught and reported

2 tests, 0 failures


real    0m1.116s
user    0m1.056s
sys     0m0.226s
```

Reply:
Nope:
```
$ time ./bin/bats --formatter bats-format-junit --filter "errors are caught" test
1..2
ok 1 halfway setup_file errors are caught and reported
ok 2 halfway teardown_file errors are caught and reported

real    0m0,853s
user    0m0,828s
sys     0m0,180s

$ time ./bin/bats --filter "errors are caught" test
 ✓ halfway setup_file errors are caught and reported 
 ✓ halfway teardown_file errors are caught and reported 

2 tests, 0 failures


real    0m0,482s
user    0m0,444s
sys     0m0,128s
```

Reply:
Just to make sure there was no misunderstanding: By 

> when tailing the output file

you mean you did `tail -f TestReport-.xml` while the above command ran, right?

Reply:
I fixed a bug where not adding `-T` when using `--formatter junit` would break the formatter due to missing timing outputs. Maybe that resolved your issue too?